docker-compose.yaml

```diff
@@ -4,8 +4,8 @@ services:
   web:
     image: skyportal/web
     ports:
-      - "9000:5000"
-      - "9001:5001"
+      - "5000:5000"
```

Comment:
Isn't this supposed to map internal 5000 to external 9000, so docker images can run on macos?

Reply:
Maybe, in which case the existing config is incorrect then? The docker ports should be <internal>:<external>, right?

Reply:
Or maybe I got it reversed, might be ...

Reply:
Can never remember :)

Reply:
yep its external -> internal, so I reversed that change and tested it